spec/models/miq_policy_spec.rb

```diff
@@ -100,6 +100,16 @@
       end
     end
 
+    describe '#miq_event_definitions' do
+      before do
+        policy.miq_policy_contents.push(FactoryGirl.create(:miq_policy_content))
+      end
+
+      it 'lists event definition' do
+        expect(policy.miq_event_definitions).to eq([event])
```

Comment:
yes you are right
but I am not sure if this comparing for activerecord object works.

@chrisarcand  are there differences between `match_array` and `eq` in this case ?